Tools-Override/tests.targets

```diff
@@ -199,15 +199,20 @@
 
     <PropertyGroup Condition="'$(UseDotNetNativeToolchain)' == 'true'">
       <ILCBuildType Condition="'$(ILCBuildType)' == ''">ret</ILCBuildType>
+      <_TestILCFolder>$(TestILCFolder)</_TestILCFolder>
+      <!--When archiving tests, ilc.exe will be located at %RUNTIME_PATH%\..\TestILC-->
+      <_TestILCFolder Condition="'$(ArchiveTests)' == 'true'">%RUNTIME_PATH%\..\TestILC</_TestILCFolder>
```

Comment:
I think it is a bad-idea to assume that the runtime payload will end up putting TestILC in the same relative path. We need to be sure that the the full payload will have the correct structure.